.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2023 Sebastian Pipping <sebastian@pipping.org>
+# Licensed under the MIT license
+
+repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.1.7
+    hooks:
+      - id: ruff
```

Comment:
```suggestion
      - id: ruff
        args: [--show-source]
```

Reply:
@jairhenrique good idea, done with next push in a minute